apps/src/aichat/views/tabs/Tabs.tsx

```diff
@@ -0,0 +1,83 @@
+import React, {useState} from 'react';
+import classNames from 'classnames';
+
+import styles from './tabs.module.scss';
+import TabPanel from './TabPanel';
+
+type Tab = {
+  title: string;
+  content: React.ReactNode;
+};
+
+interface TabsProps {
```

Comment:
nit: mind adding some comments to explain the props (mostly for the intended use of `name`)?